.github/actions/gradle/experiment-2/action.yml

```diff
@@ -26,6 +26,9 @@ inputs:
   enableGradleEnterprise:
     description: "Enables Gradle Enterprise on a project not already connected"
     required: false
+  failIfNotFullyCacheable:
+    description: "Fail the build if it is not fully cacheable"
```

Comment:
I would use the text `Terminates with exit code 3 if the build is not fully cacheable` (same as on the cmd line help text) and apply to all 5 experiments in this PR.